dev/io.openliberty.wsoc.2.2.internal_fat/fat/src/io/openliberty/wsoc22/fat/FATSuite.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2023 IBM Corporation and others.
+ * Copyright (c) 2024 IBM Corporation and others.
```

Comment:
Copyright date range should be a range: `2023, 2024` where 2023 is the date the file was created and 2024 is the latest year of a change.